<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-K6J63E8" ignored="0" name="Default test" tests="13" failures="3" timestamp="2020-10-08T23:13:07 EDT" time="138.811" errors="0">
  <testcase name="logInTest" time="2.78" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="logInTest" time="2.733" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="logInTest" time="2.813" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="customerServiceDisplay" time="0.501" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="verifiedPersonalModule" time="0.283" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;[]Personal&amp;gt; but was:&amp;lt;[selected
]Personal&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<[]Personal> but was:<[selected
]Personal>
at junit.framework.Assert.assertEquals(Assert.java:100)
at junit.framework.Assert.assertEquals(Assert.java:107)
at com.qa.wellsFargo.TestScripts.TestCaseScripts.verifiedPersonalModule(TestCaseScripts.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifiedPersonalModule -->
  <testcase name="verifiedSmallBusinessModule" time="0.412" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="verifiedCommericalModule" time="0.041" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="verifiedFinancialEduModule" time="0.414" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="verifiedAbutWellsModule" time="0.396" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="verifiedPersonalClickAbleModule" time="0.536" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="verifiedBankingAndCreditFeature" time="3.858" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;Credit Card[]&amp;gt; but was:&amp;lt;Credit Card[s]&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<Credit Card[]> but was:<Credit Card[s]>
at junit.framework.Assert.assertEquals(Assert.java:100)
at junit.framework.Assert.assertEquals(Assert.java:107)
at com.qa.wellsFargo.TestScripts.TestCaseScripts.verifiedBankingAndCreditFeature(TestCaseScripts.java:104)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifiedBankingAndCreditFeature -->
  <testcase name="verifiedIndustryExpertiseLink" time="2.523" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts"/>
  <testcase name="aboutWellsFargoHomepage" time="1.888" classname="com.qa.wellsFargo.TestScripts.TestCaseScripts">
    <failure type="junit.framework.ComparisonFailure" message="expected:&amp;lt;[a]bout Wells Fargo&amp;gt; but was:&amp;lt;[A]bout Wells Fargo&amp;gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<[a]bout Wells Fargo> but was:<[A]bout Wells Fargo>
at junit.framework.Assert.assertEquals(Assert.java:100)
at junit.framework.Assert.assertEquals(Assert.java:107)
at com.qa.wellsFargo.TestScripts.TestCaseScripts.aboutWellsFargoHomepage(TestCaseScripts.java:137)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- aboutWellsFargoHomepage -->
</testsuite> <!-- Default test -->
